course/models.py

```diff
@@ -798,10 +810,40 @@ def enroll_student(self, user):
                     user_profile=user.userprofile,
                     role=Enrollment.ENROLLMENT_ROLE.STUDENT,
                     status=Enrollment.ENROLLMENT_STATUS.ACTIVE,
+                    from_sis=from_sis,
                 )
                 return True
         return False
 
+    def enroll_from_sis(self) -> None:
+        from .sis import get_sis_configuration, StudentInfoSystem
```

Comment:
Docstring for the function would be nice. The function could have a return value, e.g., the number of students enrolled.